### ISSUE TITLE
Hospital Department concept for an Encounter

### ISSUE THREAD

**Issue by shusson** [synthetichealth/synthea#487]
Is there a concept of a department for an Encounter? I see that there's an EncounterClass but I'm looking for values like "Cardiology", "Anaesthetics", "Gastroenterology" etc.

**Comment by jawalonoski** [comment]
We haven't modeled individual specialists [yet?].

One approach, which you hint at, could be to add an optional `"specialty"` attribute on Encounter.

If that was done, all the modules would need to be reviewed and updated as appropriate to account for that information. The exporters would also need to be updated to export that data appropriately in the various supported formats.

**Comment by shusson** [comment]
Yeah makes sense. A bit of background why we want this is that a basic use case for us is
`A researcher wants to search for observations that happened in a particular department, e.g Cardiology`.

**Comment by jawalonoski** [comment]
Makes perfect sense. I'm not sure where this fits into my priorities, but I'd be happy to consider a pull request on the topic.

**Comment by shusson** [comment]
Great, we'll try and put something together.
